src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/InboundEventsListener.kt

```diff
@@ -20,6 +25,8 @@ class InboundEventsListener(
     private val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
+  @SqsListener("bvls", factory = "hmppsQueueContainerFactoryProxy")
+  @WithSpan(value = "Digital-Prison-Services-book_a_video_link_queue", kind = SpanKind.SERVER)
   fun onMessage(rawMessage: String) {
```

Comment:
^ Queue name is not right?  `farsight-devs-dev-book_a_video_link_queue` ?

Reply:
This is what gets logged in app insights, unrelated.  This is so you can query on it.